whitelist_addresses.example.csv

```diff
@@ -0,0 +1,4 @@
+stars1...
+cosmos1...
+osmo...
+terra...
```

Comment:
I think Terra uses a different derivation path, along with Desmos. So I don't think it'll work, but please double check.

Reply:
It's complicated. Terra station and the Terra Ledger app have a custom path. When you integrate Terra in Keplr via Osmosis, you get the Cosmos Hub path for Terra as well.

Reply:
`yarn run whitelist --add terra13yxhrk08qvdf5zdc9ss5mwsg5sf7zva9xrgwgc` converted `terra13yxhrk08qvdf5zdc9ss5mwsg5sf7zva9xrgwgc` to `stars13yxhrk08qvdf5zdc9ss5mwsg5sf7zva95m9npf`

also bech referenced here: https://github.com/terra-money/amino-js/blob/1faf3295f4d4b7fa491acd844900a0b859846ddc/test/bech32.test.ts

Reply:
I think the point here is that converting from non coin type 118 to `stars` address might have a different outcome when a users go to keplr and find out the address doesn't match because of the derivation path. 

I'd say it would be dangerous to do auto convert from the ones we know won't work. Or just allow the ones we know it will work (eg cosmos, osmosis, regen and others)

Reply:
This is a good idea @jhernandezb, the `toStars()` function should only convert addresses we know that use coin type 118. Can be a separate PR.